src/pages/Content/CodeGen.tsx

```diff
@@ -168,7 +172,12 @@ export function genCode(
         scriptBuilder.resize(action.width, action.height);
         break;
       case ActionType.Wheel:
-        scriptBuilder.wheel(action.deltaX, action.deltaY);
+        scriptBuilder.wheel(
+          action.deltaX,
+          action.deltaY,
+          action.pageXOffset,
```

Comment:
Need to add pageX/YOffset because Cypress scrolls in absolute units, not relative units like Playwright/Puppeteer

Reply:
Good find! I didn't know that